src/panels/developer-tools/yaml_configuration/developer-yaml-config.ts

```diff
@@ -246,6 +246,7 @@ export class DeveloperYamlConfig extends LitElement {
         .validate-log {
           white-space: pre-wrap;
           direction: ltr;
+          margin: 0 !important;
```

Comment:
Wait, does this have to be `!important` ? You should be able to use a more specific selector instead

Reply:
`pre` has some margins. In my tests it needed to be "important" to override.

Reply:
@MindFreeze retested again w/o `important` - it works in Chrome, FF, Edge.
Earlier when it did not work - was a glitch (most probably in my brains).